tests/integration/scripteditor.ts

```diff
@@ -118,6 +118,9 @@ describe('Script Editor tests', () => {
     );
     cy.get('button[title="Top"]').should('be.visible');
     cy.get('button[title="Bottom"]').should('be.visible');
+    
+    // Close editor tab
+    cy.closeTab(-1);
```

Comment:
Good catch, although I guess this will close the console output tab, maybe you'd need a second call to close the editor tab.